### ISSUE TITLE
LC0005 Duplicate diagnostic for unlimited texts

### ISSUE THREAD

**Issue by ans-bar-bm** [StefanMaron/BusinessCentral.LinterCop#639]
For texts a duplicate LC0005 diagnostic is raised when not specifying a range expression (so unlimited length):
```
codeunit 50100 MyCodeunit
{
    var
        mytext: TeXt; // duplicate diagnostic here
        mytext2: TeXt[10]; // only one diagnostic here
}
```
![LC0005](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/155651932/f22bbdca-c791-4e1d-843c-e0be61ee851c)

**Comment by Arthurvdv** [comment]
This should now be resolved in the release version of [v0.30.25](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.30.25) of the LinterCop.

**Comment by ans-bar-bm** [comment]
Did a quick test, worked for me. Thanks!
